app/src/main/java/com/example/tasky/agenda/presentation/agenda_detail/AgendaDetailViewModel.kt

```diff
@@ -121,6 +131,25 @@ class AgendaDetailViewModel(
         }
     }
 
+    private fun getReminder(reminderId: String) {
```

Comment:
Should we exposes a getReminder(id) from the repository that already returns the mapped data instead🤔?  I noticed the `ViewModel` is pulling directly from `reminderLocalDataSource` and then doing the mapping itself. That feels like work the `repository` should own -  the `ViewModel` shouldn’t need to care if the data came from local or remote, just that it gets a `Reminder` in the right shape. Right now you’ve got both `reminderRepository` and `reminderLocalDataSource` injected, which splits the responsibility and makes the ViewModel heavier than it needs to be.